.gitignore

```diff
@@ -7,4 +7,5 @@
 node_modules
 
 # npm-debug log
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+yarn-error.log
```

Comment:
Should be `#yarn-debug log` and the log should be placed under there or you change `#npm-debug log` to `npm and yarn-debug log`